canonical-web-and-design/maas-ui
# feat(ui): add the create physical interface form

## Done

- Add a form to create physical interfaces to the networking tab.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Find a machine with editable network.
- Click "Add interface" above the table.
- Fill out the details and submit. The interface should be created.

## Fixes

Fixes: #2151.
